document/CoreFormat.mdk

```diff
@@ -2281,6 +2287,17 @@ Tools that do not construct `fullDescription` in this way should provide a value
 because otherwise, the initial portion of `fullDescription` that a viewer displays where available space is limited
 might not be understandable.
 
+### `defaultLevel` property { #rule-defaultLevel }
+
+A `rule` object may contain a property named `defaultLevel` whose value shall be one of
+the strings defined for the `result.level` property (&sect;[#result-level]).
```

Comment:
Right, the subtlety here is that defaultLevel represents the severity level that is assumed for cases when a result represents an actual violation (not a pass result, and not an indication that the check did not fire because the target was not applicable). You just found a schema bug, the defaultLevel values include 'pass' and 'not applicable'. Yes, I would make this clear in the spec (what values are legal)